src/include/ipc4/base_fw.h

```diff
@@ -365,6 +365,10 @@ enum ipc4_fw_config_params {
 	IPC4_KPB_RT_SINK_COUNT = 27,
 	/* Manual control of DMI L1 state */
 	IPC4_DMI_FORCE_L1_EXIT = 28,
+	/* Intel reserved */
+	IPC4_INTEL_RESRVED_29 = 29,
+	IPC4_INTEL_RESRVED_30 = 30,
+	IPC4_INTEL_RESRVED_31 = 31,
```

Comment:
If we are using 31 we can name it using its real name.

Reply:
sure